test/jdk/com/sun/management/ThreadMXBean/VirtualThreads.java

```diff
@@ -138,7 +140,11 @@ void testGetThreadCpuTime() {
             long tid1 = vthread.threadId();
             long[] tids = new long[] { tid0, tid1 };
             long[] cpuTimes = bean.getThreadCpuTime(tids);
-            assertTrue(cpuTimes[0] >= 0L);
+            if (Thread.currentThread().isVirtual()) {
```

Comment:
tid0 is the thread ID of a platform therad. tid1 is the threadID of a virtual thread. The only change here is allow this test run with the main wrapper plugin ([CODETOOLS-7903373](https://bugs.openjdk.org/browse/CODETOOLS-7903373)), it would otherwise have to be excluded for those runs.

Reply:
I don't see any problemlist changes. Was this test failing when using the wrapper because of the lack of problemlisting?

Reply:
I added this test recently via JDK-8303242. It failed when we sync'ed up the loom repo as the test configuration there runs the tests with the jtreg main wrapper. It was trivial to fix and this avoid needing to exclude it via ProblemList-Virtual.txt. Once jtreg is promoted and there is config added to run the tests with the virtual ThreadFactory then this test will be able to run in this mode.